client.go

```diff
@@ -75,7 +75,7 @@ func (c *Client) Query(expr string) (QueryResponse, error) {
 		return nil, err
 	}
 
-	u.Path = queryPath
+	u.Path += queryPath
```

Comment:
Let's do `u.Path = strings.TrimRight(u.Path, "/") + queryPath` to be a bit cleaner.